### PR TITLE
Replace spring-security 5.7.2 dep. in core with 5.7.5.  UNF. The plugin is a different animal and so this won't get there from here.

### PR DESCRIPTION
**Pull request by jppetrakis**
Usage of jenkins core changed to 2.377.  This fixes the problematic issue with spring-security without overrides.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -34,7 +34,14 @@ dependencies {
     // or Jenkins plugin versions in Jenkins plugins that depend on this library.
     // Jenkins plugins that depend on this library should therefore version those dependencies compatibly with the versions below:
     // --rotte DEC 2019
-    implementation 'org.jenkins-ci.main:jenkins-core:2.359'
+    implementation('org.jenkins-ci.main:jenkins-core:2.359') {
+        // spring-security-web pulls in 'core' and 'crypto'.. we now replace that 5.7.2 with 5.7.5
+        // this still won't filter thru into the plugin however, which is unfortunate but this
+        // will at least also clean up annotation which depends on jenkins-common.
+        exclude group: 'org.springframework.security', module: 'spring-security-web'
+    }
```
Comment (stevebillings) [inline build.gradle:42]: I would have made the same guess here (that line 41 wouldn't be required), but, again, you may have already proven otherwise.

Comment (jppetrakis) [reply build.gradle:42]: I've seen this approach used in more than one place, and it appeared to be the "most endorsed"... at least from my search results regarding the topic.

**Review by dterrysynopsys — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (jppetrakis) [reply build.gradle]: moving to 2.377 removes the need to do this one....

Comment (jppetrakis) [reply build.gradle]: move to jekins 2.377 removes the need to play this game.

**Review by gopannabd — APPROVED** [review]

**Review by stevebillings — APPROVED** [review]

**Pull request merged**